source/includes/api/_graph.md

```diff
@@ -738,3 +738,16 @@ curl -X GET https://api.resourcewatch.org/v1/graph/query/most-liked-datasets
 	]
 }
 ```
+
+## Count dataset view
+
+Updates the view count for the corresponding dataset. 
```

Comment:
So looking at the code, it seems that the behavior is slightly different:
- It doesn't necessarily require a user (which it probably should)
- It updates the total view counter of the dataset
- If a user is present, it also updates the view counter for that dataset by that user

https://github.com/resource-watch/graph-client/blob/52176d55884077fe1852fed7c5696e8cf5b09578/app/src/routes/api/v1/graph.router.js#L457

I didn't have a change to test, but I'm leaving the link to the code above, so you can take a look and validate my logic, @pablopareja

Reply:
Oh you're right, thanks for spotting that! 👍 
I dived a bit more into the code and confirmed your assumption: https://github.com/resource-watch/graph-client/blob/52176d55884077fe1852fed7c5696e8cf5b09578/app/src/services/neo4j.service.js#L217
Let me update the docs accordingly.